pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -4725,6 +4725,22 @@ let
         };
       };
 
+      sunshaoce.risc-v = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          hash = "sha256-d6sf7is/vAiP11KdPlwbtqz3FZOvidlxj9yHHJAb5vU=";
+          name = "risc-v";
+          publisher = "sunshaoce";
+          version = "1.158.10";
```

```suggestion
          version = "1.159.1";
```
There is a new version out there.